src/client.rs

```diff
@@ -35,19 +35,42 @@ use crate::{ua, Error, Result};
 pub struct ClientBuilder(ua::Client);
 
 impl ClientBuilder {
-    /// Sets secure channel life time.
+    /// Sets (response) timeout.
     ///
     /// # Panics
     ///
     /// The given duration must be non-negative and less than 4,294,967,295 milliseconds (less than
     /// 49.7 days).
     #[must_use]
-    pub fn secure_channel_lifetime(mut self, secure_channel_lifetime: Duration) -> Self {
-        let config = unsafe { UA_Client_getConfig(self.0.as_mut_ptr()).as_mut() };
+    pub fn timeout(mut self, timeout: Duration) -> Self {
+        self.config_mut().timeout = u32::try_from(timeout.as_millis())
+            .expect("timeout (in milliseconds) should be in range of u32");
+        self
+    }
 
-        config.unwrap().secureChannelLifeTime = u32::try_from(secure_channel_lifetime.as_millis())
-            .expect("secure channel life time should be in range of u32");
+    /// Sets client description.
+    ///
+    /// The description must be internally consistent. The application URI set in the application
+    /// description must match the URI set in the certificate.
+    #[must_use]
+    pub fn client_description(mut self, client_description: ua::ApplicationDescription) -> Self {
+        client_description.move_into_raw(&mut self.config_mut().clientDescription);
+        self
+    }
 
+    /// Sets secure channel life time.
+    ///
+    /// After this life time, the channel needs to be renewed.
+    ///
+    /// # Panics
+    ///
+    /// The given duration must be non-negative and less than 4,294,967,295 milliseconds (less than
+    /// 49.7 days).
+    #[must_use]
+    pub fn secure_channel_lifetime(mut self, secure_channel_lifetime: Duration) -> Self {
```

Comment:
When following the naming conventions consistently then this method would be named `secure_channel_life_time`. The term _life time_ in two words is also used in the docs.